src/components/generic/OptionsList/OptionsList.jsx

```diff
@@ -23,14 +23,17 @@ export default function OptionsList({
 }) {
   const [cssClasses, sc] = webexComponentClasses('options-list', className);
 
+  const [currentOptionIndex, setCurrentOptionIndex] = useState();
+
   return (
     <div className={cssClasses}>
       <ul role="menu" className={sc('list')}>
-        {options.map((option) => (
+        {options.map((option, index) => (
           <li
-            className={sc('option')}
+            className={`${sc('option')} ${(currentOptionIndex === index) && sc('option--focused')}`}
             key={option.value}
             onClick={() => onSelect(option)}
+            onMouseEnter={() => setCurrentOptionIndex(index)}
             aria-hidden="true"
```

Comment:
remove this pls

Reply:
I will delete when I add onKeyDown event for enter because it would give error if I only remove that aria-hidden